### PR TITLE
Feature/SDL passenger mode

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
Provides the information for #[2134](https://github.com/SmartDeviceLink/sdl_core/issues/2134)

This PR is **[ready]** for review.

### Summary
Update OnDriverDistraction index

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: docs/UI/OnDriverDistraction/index.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ Purpose
 
 ### Notification
 
+!!! must
+
+1. Inform SDL whenever the driver distraction mode is activated/deactivated on HMI.
+
+!!!
+
+Driver distraction rules may be specific to contry/area, so it depends on HMI when to trigger activate/deactivate states.
```
Comment (jacobkeeler) [inline docs/UI/OnDriverDistraction/index.md]: `contry` -> `country`

File: docs/UI/OnDriverDistraction/index.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ Purpose
...
+
+_Note:_
```
Comment (jacobkeeler) [inline docs/UI/OnDriverDistraction/index.md]: Use `!!! NOTE` block

File: docs/UI/OnDriverDistraction/index.md (hunk 1 above)
Comment (KhrystynaDubovyk) [reply docs/UI/OnDriverDistraction/index.md]: thank you, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/137/commits/3efc9161566c0c35b6c4bdaf00827bd0914895d0

File: docs/UI/OnDriverDistraction/index.md (hunk 2 above)
Comment (KhrystynaDubovyk) [reply docs/UI/OnDriverDistraction/index.md]: thank you, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/137/commits/3efc9161566c0c35b6c4bdaf00827bd0914895d0

**Comment by KhrystynaDubovyk** [comment]
the source file for the included diagram is added in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/137/commits/a4da43d9f36f0a36938816c344c70cd6f5329713

**Comment by IGapchuk** [comment]
@jacobkeeler @JackLivio Please be informed that we've added new commits with updates according to next revisions of the proposals: https://github.com/smartdevicelink/sdl_evolution/pull/766 and https://github.com/smartdevicelink/sdl_evolution/pull/775. Please take a llok at next commits: 680c1a82f43f4dac64be1c0cce7b718bc7deef48, 3ebb2ffa2285ed99a1992666882eceff1c2e047b

**Comment by mrapitis** [comment]
@theresalech Ford has reviewed and approved this PR.

**Pull request merged**